packages/api/src/makeMergedSchema/makeMergedSchema.ts

```diff
@@ -155,7 +155,7 @@ export const makeMergedSchema = ({
   services: Services
   schemaDirectives?: IExecutableSchemaDefinition['schemaDirectives']
 }) => {
-  const typeDefs = mergeTypes(
+  const typeDefs = mergeTypeDefs(
     [rootSchema.schema, ...Object.values(schemas).map(({ schema }) => schema)],
     { all: true }
```

Comment:
Fixing the error in the line above gives us same error in the following line `No overload matches this call.`